src/hexo/themes/sonarwhal/source/js/scanner-submit.js

```diff
@@ -408,6 +408,13 @@
         }
     };
 
+    var updateStatus = function (status) {
+        var statusElement = document.querySelector('.scan-overview__status');
+
+        statusElement.textContent = status;
+        statusElement.removeAttribute('id');
```

Comment:
Why do you remove the id?

Reply:
@molant Because some blinking effects are added to the text through the id (`status-analyzing`), and that effect should be removed for `finished` and `error`. Alternatively I can change the id to `status-finished` or `status-error` if preferred.

Reply:
Continues in https://github.com/sonarwhal/sonarwhal.com/pull/242#discussion_r145832111